linux_os/guide/system/network/network-nftables/nftables_rules_permanent/tests/invalid_top_level_config_file.fail.sh

```diff
@@ -0,0 +1,3 @@
+# platform = sle15
```

Comment:
Nitpick: You don't have to specify a platform in the test scenario. The rule is limited only to sle15 in the prodtype. It won't be run on other platforms anway.